public/app/plugins/datasource/grafana/datasource.ts

```diff
@@ -135,14 +134,14 @@ export class GrafanaDatasource extends DataSourceWithBackend<GrafanaQuery> {
           buffer.maxDelta = request.range.to.valueOf() - request.range.from.valueOf();
         }
 
-        results.push(
-          getGrafanaLiveSrv().getDataStream({
-            key: `${request.requestId}.${counter++}`,
-            addr: addr!,
-            filter,
-            buffer,
-          })
-        );
+        const opts = {
+          key: `live-${target.refId}-`,
```

Comment:
should this also change here? https://github.com/grafana/grafana/blob/963fffeef10cdad8276e46c6a20f2a13b0491c31/public/app/features/live/centrifuge/service.ts#L193